chainermn/communicators/non_cuda_aware_communicator.py

```diff
@@ -16,14 +16,16 @@ def __init__(self, mpi_comm):
         self.cpu_buffer_a = _memory_utility.HostPinnedMemory()
         self.cpu_buffer_b = _memory_utility.HostPinnedMemory()
 
-    def broadcast_data(self, model):
+    def bcast_data(self, model):
         for _, param in sorted(model.namedparams()):
             data = param.data
             tmp_cpu = chainer.cuda.to_cpu(data)
             self.mpi_comm.Bcast(tmp_cpu)
             tmp_gpu = chainer.cuda.to_gpu(tmp_cpu)
             data[:] = tmp_gpu
 
+    broadcast_data = bcast_data
```

Comment:
I think it is a bit annoying to have `broadcast_data = bcast_data` in all concrete communicator definitions. Let us call `bcast_data` from `broadcast_data` in `CommunicatorBase` and remove them.